osu.Game/Overlays/MusicController.cs

```diff
@@ -75,7 +75,7 @@ public void ChangeBeatmapSetPosition(BeatmapSetInfo beatmapSetInfo, int index)
         /// <summary>
         /// Returns whether the current beatmap track is playing.
         /// </summary>
-        public bool IsPlaying => beatmap.Value?.Track.IsRunning ?? false;
+        public bool IsPlaying => beatmap?.Value?.Track.IsRunning ?? false;
```

Comment:
Why not use `current` instead?
```suggestion
        public bool IsPlaying => current?.Track.IsRunning ?? false;
```

Reply:
Sure